Migrations.cs

```diff
@@ -512,5 +513,22 @@ public int UpdateFrom11()
 
             return 12;
         }
+
+        public int UpdateFrom12()
+        {
+            _contentDefinitionManager.AlterPartDefinition(nameof(Heading), builder => builder
+                .WithField(nameof(Heading.Emphasize), field => field
+                    .OfType(nameof(Fields.Values.Fields.ValuesField))
+                    .WithSettings(new ValuesFieldSettings
+                    {
+                        EmptyMessage = "Currently no text will be emphasized.",
+                        Hint = "Content within the heading text that should be given emphasis.",
+                        NewItemPlaceholder = "Text matching is case sensitive."
+
+                    })
+                    .WithDisplayName("Emphasize")
+                    .WithPosition("5")));
+            return 13;

```

Comment:
Knock this down a line to have an empty line between the migration code and the return.